### PR TITLE
fix connect app issue 2741 - [Team Management 2.0] Handle the case when email entered to be invited is already topcoder user

### PR DESCRIPTION
**Pull request by maxceem**
Fix for Connect App issue https://github.com/appirio-tech/connect-app/issues/2741 - [Team Management 2.0] Handle the case when email entered to be invited is already topcoder user

### CODE REVIEW

File: src/routes/projectMemberInvites/create.spec.js [hunk 1]
```diff
@@ -275,6 +277,57 @@ describe('Project Member Invite create', () => {
...
+        .expect(201)
+        .end((err, res) => {
+          if (err) {
+            done(err);
+          } else {
+            const resJson = res.body.result.content[0];
+            should.exist(resJson);
+            resJson.role.should.equal('customer');
+            resJson.projectId.should.equal(project2.id);
+            resJson.userId.should.equal(12345);
```
Comment (RishiRajSahu) [inline src/routes/projectMemberInvites/create.spec.js]: @maxceem can we add check for email Id as well here ?

File: src/routes/projectMemberInvites/create.js [hunk 2]
```diff
@@ -118,9 +118,35 @@ module.exports = [
...
+            const nonExistentUserEmails = invite.emails.filter(inviteEmail =>
+              !_.find(existentUsers, { email: inviteEmail }),
+            );
+
+            // remove invites for users that are invited already
+            _.remove(existentUserIds, userId => _.some(invites, i => i.userId === userId));
+            existentUserIds.forEach((userId) => {
+              const dataNew = _.clone(data);
+              _.assign(dataNew, {
+                userId,
```
[the remark this answers is not in the document]
Comment (maxceem) [reply src/routes/projectMemberInvites/create.js]: As we create a new object `const dataNew = _.clone(data);` and modify only `dataNew` means we don't modify `data`. So we don't need to set `data.userId = null;` after.

File: src/routes/projectMemberInvites/create.spec.js (hunk 1 above)
Comment (maxceem) [reply src/routes/projectMemberInvites/create.spec.js]: Added check email.

**Comment by maxceem** [comment]
@RishiRajSahu added the check for email in unit tests. Lint is fine.

**Pull request merged**